server/src/main/java/io/crate/execution/engine/export/FileOutput.java

```diff
@@ -21,15 +21,19 @@
 
 package io.crate.execution.engine.export;
 
+import io.crate.execution.dsl.projection.WriterProjection;
+
 import java.io.IOException;
 import java.io.OutputStream;
+import java.net.URI;
+import java.util.concurrent.Executor;
 
-public abstract class Output {
+public interface FileOutput {
 
     /**
      * calling this method creates & acquires an OutputStream which must be closed by the caller if it is no longer needed
      *
      * @throws IOException in case the Output can't be created (e.g. due to file permission errors or something like that)
      */
-    public abstract OutputStream acquireOutputStream() throws IOException;
+    OutputStream getStream(Executor executor, URI uri, WriterProjection.CompressionType compressionType) throws IOException;
```

Comment:
I think we had the convention of using `acquire` to indicate if something has to be released by the caller.
At least it is used in a couple other places that way - but no strong opinion on it. I think we get alerts of unclosed `Closeable` instances anyways

Reply:
Thanks for the info, I blindly refactored it to match `FileInput`. Will revert this.